src/middlewared/middlewared/alert/source/m_series_nvdimm_firmware_freebsd.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding=utf-8 -*-
+import logging
+
+from middlewared.alert.base import Alert, AlertClass, AlertCategory, AlertLevel, AlertSource
+from middlewared.service_exception import CallError
+
+logger = logging.getLogger(__name__)
+
+
+class TrueNASMNVDIMMFirmwareVersionAlertClass(AlertClass):
+    category = AlertCategory.HARDWARE
+    level = AlertLevel.EMERGENCY
+    title = "Invalid NVDIMM Firmware Version"
+    text = (
+        "Your NVDIMM controller %(index)d is using an invalid firmware version %(version)s which can cause data loss."
+        "Please contact support immediately."
```

Comment:
@Mrt134 are we OK with this text?

@wmohanlon Thoughts on this one? You OK with that wording? And the level of this alert being emergency?

Reply:
@Mrt134 @wmohanlon ping

Reply:
Same thought as other comment -- could we leave out "immediately" ?